extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -31,6 +31,16 @@ import {toggle} from '../../../src/style';
 
 const TAG = 'amp-consent-ui';
 
+// Classes for consent UI
+const consentUiClasses = {
+  fullscreen: 'i-amphtml-consent-ui-fullscreen',
+  iframeActive: 'consent-iframe-active',
```

Comment:
Let's sync offline on the class name.

Reply:
I agree. Was trying to follow your example for class names. But I think it would be good to go over them again 😄